var/spack/repos/builtin/packages/openfast/package.py

```diff
@@ -79,6 +79,13 @@ def cmake_args(self):
                 'ON' if '+cxx' in spec else 'OFF'),
         ])
 
+        # Make sure we use Spack's blas/lapack:
+        blas_libs = spec['blas'].libs + spec['lapack'].libs
```

Comment:
Yes, but please change the order, lapack links against Blas (in case they are different)